map/map_loader/README.md

```diff
@@ -89,3 +93,12 @@ lanelet2_map_visualization visualizes autoware_auto_mapping_msgs/HADMapBin messa
 ### Published Topics
 
 - ~output/lanelet2_map_marker (visualization_msgs/MarkerArray) : visualization messages for RViz
+
+### Parameters
+
+| Name                        | Type   | Description                                                  | Default value |
+| :-------------------------- | :----- | :----------------------------------------------------------- | :------------ |
+| lanelet2_map_projector_type | string | The type of the map projector using, can be MGRS, UTM, local | MGRS          |
+| latitude                    | double | Latitude of map_origin, only using in UTM map projector      | 0.0           |
+| longitude                   | double | A flag to enable partial pointcloud map server               | 0.0           |
```

Comment:
This seems to be wrong explanation.

```suggestion
| longitude                   | double |  Longitude of map_origin, only using in UTM map projector               | 0.0           |
```

Reply:
Could you also add `lanelet2_map_path` to the table as well? (I know this is not related to this PR, but it would be nice to add it to make it consistent with other ROS parameters)

Reply:
Thanks, it is updated at
[7a28b37](https://github.com/autowarefoundation/autoware.universe/pull/3492/commits/7a28b379b6f20c886da413eeea2e288bd36408a3)